src/Concerns/ManagesUsageBilling.php

```diff
@@ -0,0 +1,79 @@
+<?php
+
+namespace Laravel\Cashier\Concerns;
+
+use Illuminate\Support\Collection;
+use Stripe\Billing\MeterEvent;
+
+trait ManagesUsageBilling
+{
+    /**
+     * Get all the defined billing meters.
+     *
+     * @param  array  $options
+     * @param  array  $requestOptions
+     * @return \Illuminate\Support\Collection
+     */
+    public function meters(array $options = [], array $requestOptions = []): Collection
+    {
+        return new Collection($this->stripe()->billing->meters->all($options, $requestOptions)->data);
+    }
+
+    /**
+     * Report usage for a metered product.
+     *
+     * @param  string  $meter
+     * @param  int  $quantity
+     * @param  string|null  $price
+     * @param  array  $options
+     * @param  array  $requestOptions
+     * @return \Stripe\Billing\MeterEvent
+     */
+    public function reportMeterEvent(
+        string $meter,
+        int $quantity = 1,
+        array $options = [],
+        array $requestOptions = []
+    ): MeterEvent {
+        $this->assertCustomerExists();
+
+        return $this->stripe()->billing->meterEvents->create([
+            'event_name' => $meter,
+            'payload' => [
+                'value' => $quantity,
+                'stripe_customer_id' => $this->stripeId(),
+            ],
+            ...$options,
+        ], $requestOptions);
+    }
+
+    /**
+     * Get the usage records for a meter using its ID.
+     *
+     * @param  string  $meterId
+     * @param  array  $options
+     * @param  array  $requestOptions
+     * @return \Illuminate\Support\Collection
+     */
+    public function meterEventSummaries(string $meterId, array $options = [], array $requestOptions = []): Collection
+    {
+        $this->assertCustomerExists();
+
+        $startTime = $options['start_time'] ?? 1;
+
+        $endTime = $options['end_time'] ?? time();
```

Comment:
Would these almost always be passed? If so, should we just make them arguments?

Reply:
Hi Taylor, I think so, it's required as shown here https://docs.stripe.com/api/usage-record-summary/object

So yes, I think they would be better suited as arguments to the function